index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="stylesheet.css" />
+  </head>
+  <body>
+    <nav class="nav_links">
+      <img src="geo1.png" alt="cool logo" />
+      <div class="burger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+      </div>
+    </nav>
```

Comment:
- [ ] Kindly implement the changes requested, you can use the suggested code by the reviewer,
```suggestion
    <nav class="nav_links">
      <img src="geo1.png" alt="cool logo" />
      <div class="burger">
        <div class="line1"></div>
        <div class="line2"></div>
        <div class="line3"></div>
      </div>
    </nav>
```
    <header>
      <a href="#">
        <img src="geo1.png" alt="cool logo" />
      </a>
      <nav class="nav_links">
        <div class="burger">
          <div class="line1"></div>
          <div class="line2"></div>
          <div class="line3"></div>
        </div>
      </nav>
    </header>

Reply:
done